test/flutter_map_test.dart

```diff
@@ -223,7 +215,12 @@ class TestRebuildsApp extends StatefulWidget {
 class _TestRebuildsAppState extends State<TestRebuildsApp> {
   MapController _mapController = MapController();
   Crs _crs = const Epsg3857();
-  int _interactiveFlags = InteractiveFlag.all;
+
+  /// double tap gestures delay the tap gestures, disable them here
+  EnabledGestures _interactiveFlags = const EnabledGestures.all(
```

Comment:
As above, this is another indicator that we need better categorization here.

Reply:
discussion merged with https://github.com/fleaflet/flutter_map/pull/1733#discussion_r1428964013

Reply:
The other conversation discussed the grouping of gestures by it's affect on `MapCamera` (zoom, move, rotate). This conversation means to have a categorization for gesture types like double taps.

It would be possible to add another constructor `EnabledGestures.byCategories(List<enum GestureCategory>)` with zooming, rotating, dragging, doubleTapping as fields. However it's unclear to me if a reasonable amount of users would use this.

Reply:
marking this conversation as resolved as https://github.com/fleaflet/flutter_map/pull/1733#discussion_r1428964013 is resolved.